app/code/Magento/CatalogImportExport/Model/Import/Product.php

```diff
@@ -159,6 +159,11 @@ class Product extends \Magento\ImportExport\Model\Import\Entity\AbstractEntity
      */
     const URL_KEY = 'url_key';
 
+    /**
+    * SKU max length
+    */
+    const DB_MAX_SKU_LENGTH = 65;
```

Comment:
You introduced the constant in the class marked with `@api` that is backward incompatible (https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/). Additionally the size of `sku` column is 64, not 65. Please reimplement the solution.

Reply:
> Also we want the fixes to be delivered to 2.3 and then backported to 2.2. Please create PR to 2.3
Hello, I have created a new pull request #18639 as per your suggestion based on 2.3 develop branch. Please kindly review it.

Reply:
Done. Thank you for urgent reaction :)